src/cli/cli.cpp

```diff
@@ -536,6 +536,57 @@ template <> otError Interpreter::Process<Cmd("br")>(Arg aArgs[])
         OutputIp6PrefixLine(nat64Prefix);
     }
 #endif // OPENTHREAD_CONFIG_BORDER_ROUTING_NAT64_ENABLE
+    /**
+     * @cli br omravdprf [high\med\low]
+     *
+     * @code
+     * br omradvprf
+     * med
+     * Done
+     * @endcode
+     *
+     * @cparam br omradvprf [@ca{high}|@ca{med}|@ca{low}]
+     *
+     * @code
+     * br omradvprf low
+     * Done
+     * @endcode
+     *
+     * @par api_copy
+     * #otBorderRoutingGetOmrPrefixAdvertisementPreference
+     *
+     */
```

Comment:
Will this be translated to public documentations?

Reply:
Yes. This is the new cli documentation model (which will generate nice documentation site on "openthread.io"). 
I still added the documentation  using old style in `README.md` as well.